drivers/scheduler/sidecars/scripts/sysbench.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+trap '' INT
+trap '' QUIT
+trap '' STOP
+until mysql -h ${MYSQL_HOST} -u root -p${MYSQL_ROOT_PASSWORD} -e "CREATE SCHEMA IF NOT EXISTS sbtest ; CREATE USER IF NOT EXISTS sysbench@'%' IDENTIFIED BY 'password'; GRANT ALL PRIVILEGES ON sbtest.* to sysbench@'%';"
+do
+  echo "failed to create schema and user. retrying..."
+  sleep 2
+done
+while :
+do
+    sysbench --db-driver=mysql --oltp-table-size=1000000 --oltp-tables-count=24 --threads=4 --mysql-host=${MYSQL_HOST} --mysql-port=3306 --mysql-user=sysbench --mysql-password=password /usr/share/sysbench/tests/include/oltp_legacy/parallel_prepare.lua prepare
```

Comment:
all of these parameters need to be configurable

Reply:
https://portworx.atlassian.net/browse/PTX-1535